### DISCUSSION TITLE
没有URI资源权限

### DISCUSSION THREAD

**Discussion by nizpxox** [zuihou/zuihou-admin-cloud#301] [01.常见问题]
![image](https://github.com/dromara/lamp-cloud/assets/141626156/f5fa7235-96a2-4370-9378-e0a828f0c69e)
![image](https://github.com/dromara/lamp-cloud/assets/141626156/c75a00c5-0d92-4f7a-9295-5398c4838d0c)
请求头都加了，他这个接口传的employeeId也都有，为什么报错，测试好多都是没有URI资源权限，之前有一个解决了是参数名写错了，这次都是复制粘贴的。
我只传一个ID也是报错，这个ID是我设置的员工Id 相当于租户管理员，权限都有，
我传了我base1表中的所有员工Id都显示没有该URI的权限
这个是只传Id的
![image](https://github.com/dromara/lamp-cloud/assets/141626156/c9e1b310-3ab5-4157-a37e-3bbdfd185dcf)

**Comment by zuihou** [comment]
<img width="2265" alt="image" src="https://github.com/dromara/lamp-cloud/assets/8500410/177879a1-a119-4293-bb64-54a136c7afc8">

注意4个请求头，注意大小写

**Reply by zuihou** [reply]
注意Token是否有权限
